coral/src/app/pages/topics/index.tsx

```diff
@@ -1,12 +1,15 @@
 import BrowseTopics from "src/app/features/browse-topics/BrowseTopics";
-import { Flexbox, PageHeader } from "@aivenio/design-system";
+import { PageHeader } from "@aivenio/design-system";
+import Layout from "src/app/layout/Layout";
 
 const Topics = () => {
   return (
-    <Flexbox direction={"column"} rowGap={"l2"}>
-      <PageHeader title={"Browse all topics"} />
-      <BrowseTopics />
-    </Flexbox>
+    <Layout>
+      <>
```

Comment:
Nitpick: why do we have a fragment here? `Layout` wrapping those two components should be enough?

Reply:
`children` in Layout is of type `JSX.Element` from my first iteration of this. 
But I think there's no downside to change the type to `ReactNode`? 🤔

Reply:
Ah, I'm sorry I missed this! I would say that requiring a single child component for this `Layout` component might be a little  strict, and I can't think of an objective reason for it (but please illuminate me if there is one!).

I think `ReactNode` makes sense, as it is what children are typed as in the `PropsWithChildren` type from React (except that it's actually `children?: ReactNode | undefined;`, and I'm pretty sure we don't want `undefined` in `Layout` for `children`).